spring-cloud-dataflow-configuration-metadata/src/test/java/org/springframework/cloud/dataflow/container/registry/DefaultContainerImageMetadataResolverTest.java

```diff
@@ -210,6 +211,25 @@ public void getImageLabelsWithInvalidLabels() throws JsonProcessingException {
 		assertThat(labels).isEmpty();
 	}
 
+	@Test
+	public void getImageLabelsWithMixedOCIResponses() throws JsonProcessingException {
```

Comment:
Probably want to have a negative test in case the config is missing.

Reply:
An exception is thrown if the config is missing.

Reply:
Wouldn't we want a test for that?

Reply:
If happens when people push OCI and Docker containers to the same registry. With Quay it is the case. Data flow configuration provides for specifying a manifest type to accept but when the registry has both type it fails on containers of the other type. So users should not configure a manifest type and we will use docker manifest type first and if that fails we use OCI manifest type to obtain the sha. Then the labels are retrieved using the sha.